sdk/web-pubsub/web-pubsub-client/package.json

```diff
@@ -16,14 +16,15 @@
     "./dist-esm/src/websocket/websocketClient.js": "./dist-esm/src/websocket/websocketClient.browser.js",
     "events": "eventemitter3"
   },
+  "jsdelivr": "./dist-browser/index.browser.js",
```

Comment:
does `jsdelivr` support ESM browser imports? We're moving to that pattern across all packages and I think it'd be better than pulling in webpack in a one-off way here. Webpack has a huge dependency graph.

Reply:
Is this work currently in progress? Also, are there any samples output that we could look at? Whether the output can be directly referenced by in browser. I'd like to understand if there's anything specific that our service team needs to do to support this transition.
Additionally, regarding the ESM version that's being generated, will the original dependencies be treated as external dependencies and not be bundled directly into the output file?